### ISSUE TITLE
TODO?

### ISSUE THREAD

**Issue by wxguy** [leifdenby/tephigram_python#2]
Any plan to implement todo as you mentioned in your project page? Again this would be nice addition to the project.

**Comment by leifdenby** [comment]
Yes, I've actually implemented LCL and LFC already. When I finish the cleanup I mentioned on #1 I'll put these in too.

**Comment by wxguy** [comment]
Thanks. Looking forward to use the fresh code...

**Comment by mqbprjb5** [comment]
Hi. Any updates for the TODO points? Cheers

**Comment by leifdenby** [comment]
I've finally managed to add my code to do the parcel integration to compute CIN and CAPE. Hope this is useful :)

**Comment by wxguy** [comment]
> 
> 
> I've finally managed to add my code to do the parcel integration to compute CIN and CAPE. Hope this is useful :)

Yes indeed. Excellent and please continue your good work.
